### PR TITLE
Added a place holder for image in profile page

### PR DESCRIPTION
**Pull request by VillanuevaK**
I'm not sure how to test if the image appears on the profile page without using an actual picture. I tried using an array of bytes, however nothing changed.

### CODE REVIEW

File: src/main/java/codeu/model/data/Media.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+/**
+ * Class representing media (a picture, video, or sound byte).
+ * The media is uploaded by Users.
+ */
+public class Media {
+  private final UUID id;
+  private final UUID owner;
+  private final Instant creation;
+  private final String title;
+  private final Blob content;
```
Comment (yufangqq) [inline src/main/java/codeu/model/data/Media.java]: Consider adding a ENUM private field 'type',

public enum MediaType {
    JPG, PNG, MP4
}

Comment (yufangqq) [reply src/main/java/codeu/model/data/Media.java]: Or maybe something like

public enum MediaType {
  PROFILE_PICTURE,
  CHAT_IMAGE
}

File: src/main/java/codeu/model/store/persistence/PersistentDataStore.java [hunk 2]
```diff
@@ -150,6 +152,42 @@ public PersistentDataStore() {
...
+   *
+   * @throws PersistentDataStoreException if an error was detected during the load from the
+   *     Datastore service
+   */
+  public List<Media> loadMedia() throws PersistentDataStoreException {
+
+    List<Media> media = new ArrayList<>();
+
+    // Retrieve all conversations from the datastore.
+    Query query = new Query("chat-media").addSort("creation_time", SortDirection.ASCENDING);
```
Comment (yufangqq) [inline src/main/java/codeu/model/store/persistence/PersistentDataStore.java:166]: One thing to think about is that there could be many different types of media types (e.g., jpg, png), and media purposes (profile pictures vs other usages). Then given a piece of media in the chat-media table, how do I know what it is used for?

**Review by yufangqq — APPROVED** [review]

File: src/main/java/codeu/model/store/persistence/PersistentDataStore.java (hunk 2 above)
Comment (VillanuevaK) [reply src/main/java/codeu/model/store/persistence/PersistentDataStore.java:166]: Could I somehow use enums in the media class for this?

Comment (yufangqq) [reply src/main/java/codeu/model/store/persistence/PersistentDataStore.java:166]: Sure, what is your implementation to support this is up to you.

**Pull request merged**